services/core-web/src/components/dashboard/majorProjectHomePage/MajorProjectTable.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { FC } from "react";
+import { Link } from "react-router-dom";
+import { Button, Col, Row } from "antd";
+import { uniqBy, flattenDeep } from "lodash";
+import * as Strings from "@mds/common/constants/strings";
+import {
+  PROJECT_SUMMARY_STATUS_CODES,
+  MAJOR_MINE_APPLICATION_AND_IRT_STATUS_CODES,
+  PROJECT_DECISION_PACKAGE_STATUS_CODES
+} from "@mds/common/constants/enums";
+import { formatDate, dateSorter } from "@common/utils/helpers";
```

Comment:
this one I'd recommend updating the import to be from @mds/common

Reply:
The import has been changed to be from @mds/common now